UnityProject/Assets/Scripts/Items/ItemAttributesV2.cs

```diff
@@ -247,7 +247,14 @@ public bool HasAnyTrait(IEnumerable<ItemTrait> expectedTraits)
 		/// <returns></returns>
 		public bool HasAllTraits(IEnumerable<ItemTrait> expectedTraits)
 		{
-			return traits.All(expectedTraits.Contains);
```

Comment:
why this change?

Reply:
it required having the exact same list of ItemTrait for some reason